org.zenframework.z8.server/src/main/java/org/zenframework/z8/server/base/table/system/Requests.java

```diff
@@ -33,6 +40,8 @@ public CLASS(IObject container) {
 			setJavaClass(Requests.class);
 			setName(TableName);
 			setDisplayName(displayNames.Title);
+			Optional.ofNullable(apiAttrs.Title)
+					.ifPresent(attrVal -> setAttribute("APIDescription", attrVal));

```

Comment:
Write without lambda (->), declare APIDescription constant in org.zenframework.z8.server.json.Json